src/components/homePage/Search/__tests__/FilmList.test.tsx

```diff
@@ -0,0 +1,176 @@
+    },
+    {
+      Title: "The Lego Batman Movie",
+      Year: "2017",
+      imdbID: "tt4116284",
+      Type: "movie",
+      Poster:
+        "https://m.media-amazon.com/images/M/MV5BMTcyNTEyOTY0M15BMl5BanBnXkFtZTgwOTAyNzU3MDI@._V1_SX300.jpg",
+    },
+    {
+      Title: "Batman: The Animated Series",
+      Year: "1992–1995",
+      imdbID: "tt0103359",
+      Type: "series",
+      Poster:
+        "https://m.media-amazon.com/images/M/MV5BOTM3MTRkZjQtYjBkMy00YWE1LTkxOTQtNDQyNGY0YjYzNzAzXkEyXkFqcGdeQXVyOTgwMzk1MTA@._V1_SX300.jpg",
+    },
+    {
+      Title: "Batman v Superman: Dawn of Justice (Ultimate Edition)",
+      Year: "2016",
+      imdbID: "tt18689424",
+      Type: "movie",
+      Poster:
+        "https://m.media-amazon.com/images/M/MV5BOTRlNWQwM2ItNjkyZC00MGI3LThkYjktZmE5N2FlMzcyNTIyXkEyXkFqcGdeQXVyMTEyNzgwMDUw._V1_SX300.jpg",
+    },
+  ];
+  test("component is render", () => {
+    //Arange
+    const setCurrentImageMock = jest.fn();
+    const removeCurrentImageMock = jest.fn();
+    const prevPageMock = jest.fn();
+    const nextPageMock = jest.fn();
+    const currentImage =
+      "https://m.media-amazon.com/images/M/MV5BOTRlNWQwM2ItNjkyZC00MGI3LThkYjktZmE5N2FlMzcyNTIyXkEyXkFqcGdeQXVyMTEyNzgwMDUw._V1_SX300.jpg";
+
+    //Act
+    const tree = renderer
+      .create(
+        <BrowserRouter>
+          <FilmList
+            filmList={filmList}
+            isFetching={false}
+            currentImage={currentImage}
+            setCurrentImage={setCurrentImageMock}
+            removeCurrentImage={removeCurrentImageMock}
+            prevPage={prevPageMock}
+            nextPage={nextPageMock}
+            filmsCount={559}
+            pageNumber={1}
+          />
+        </BrowserRouter>
+      )
+      .toJSON();
+
+    //Assert
+    expect(tree).toMatchSnapshot();
+  });
+
+  test("pagination is render and working", () => {
+    //Arange
+    const setCurrentImageMock = jest.fn();
+    const removeCurrentImageMock = jest.fn();
+    const prevPageMock = jest.fn();
+    const nextPageMock = jest.fn();
+
+    //act
+    render(
+      <BrowserRouter>
+        <FilmList
+          filmList={filmList}
+          isFetching={false}
+          currentImage={""}
+          setCurrentImage={setCurrentImageMock}
+          removeCurrentImage={removeCurrentImageMock}
+          prevPage={prevPageMock}
+          nextPage={nextPageMock}
+          filmsCount={250}
+          pageNumber={1}
+        />
+      </BrowserRouter>
+    );
+    const navArrows = screen.getAllByRole("button");
+    fireEvent.click(navArrows[0]);
+    fireEvent.click(navArrows[1]);
+
+    //Assert
+    expect(navArrows[0]).toBeInTheDocument();
+    expect(navArrows[1]).toBeInTheDocument();
+    expect(prevPageMock).toHaveBeenCalled();
+    expect(nextPageMock).toHaveBeenCalled();
+  });
+
+  test("pagination is not render", () => {
```

Comment:
"Pagination not renders" or "Pagination is disabled"

Reply:
fixed